utils/generate_types/test/test.ts

```diff
@@ -323,6 +323,13 @@ playwright.chromium.launch().then(async browser => {
   console.log(await resultHandle.jsonValue());
   await resultHandle.dispose();
 
+  // evaluteHandle with two different return types
+  {
+    const handle = await page.evaluateHandle(() => '' as string | number);
```

Comment:
Let's add a case where with `as HTMLInputElement | HTMLTextAreaElement` and check that result is an `ElementHandle<HTMLInputElement | HTMLTextAreaElement>`?